src/main/scala/nl.knaw.dans.easy.solr4files/package.scala

```diff
@@ -119,8 +119,8 @@ package object solr4files extends DebugEnhancedLogging {
   }
 
   implicit class RichParams(val left: scalatra.Params) extends AnyVal {
-    def asString: String = if (left.isEmpty) "no params at all"
-                           else "params " + left.mkString("[", ",", "]")
+    def asString: String = if (left.size == 0) "no params at all"
+                           else "params " + left.toMap.mkString("[", ",", "]")
```

Comment:
why are these changes necessary? Tried without these and it still builds...

Reply:
This change was necessary when trying to a newer milestone version of scalatra. But since the final parent POM only contains the latest stable version, these changes are indeed no longer needed.